process/rating/peerHonesty/peerScore_test.go

```diff
@@ -0,0 +1,418 @@
+package peerHonesty
```

Comment:
:+1:  Tests cover everything with the exception of impossible cases (bad casts etc.).